### DISCUSSION TITLE
Has been working fine on my Pi4 for a while. Will not search on my pi 5

### DISCUSSION THREAD

**Discussion by liukonen** [searxng/searxng#4088] [Q&A]
I recently got a Raspberry Pi 5 box. I don't recall the steps I took to get it running on the 4, but I still have my settings.yml, and docker compose from the 4. Migrated to the 5, and all search results resulted in a time-out. I tried adjusting timeout settings and the like to no avail. 

Error reported on a search call
searxng  | 2024-12-03 02:40:54,960 ERROR:searx: call to ResultContainer.add_unresponsive_engine after ResultContainer.close
searxng  | 2024-12-03 02:40:54,960 ERROR:searx.engines.arxiv: HTTP requests timeout (search duration : 5.007252454990521 s, timeout: 5.0 s) : ConnectTimeout
searxng  | 2024-12-03 02:40:54,970 ERROR:searx: call to ResultContainer.add_unresponsive_engine after ResultContainer.close
searxng  | 2024-12-03 02:40:54,970 ERROR:searx.engines.pubmed: HTTP requests timeout (search duration : 5.017387638799846 s, timeout: 5.0 s) : ConnectTimeout
searxng  | 2024-12-03 02:40:59,381 ERROR:searx.engines.docker hub: exception : 'NoneType' object is not iterable
searxng  | Traceback (most recent call last):
searxng  |   File "/usr/local/searxng/searx/search/processors/online.py", line 160, in search
searxng  |     search_results = self._search_basic(query, params)
searxng  |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
searxng  |   File "/usr/local/searxng/searx/search/processors/online.py", line 148, in _search_basic
searxng  |     return self.engine.response(response)
searxng  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
searxng  |   File "/usr/local/searxng/searx/engines/docker_hub.py", line 41, in response
searxng  |     for item in body.get("summaries", []):
searxng  |                 ^^^^^^^^^^^^^^^^^^^^^^^^^
searxng  | TypeError: 'NoneType' object is not iterable

Tried with both a redis cache, and with redis turned off (pi4 does not hit a redis instance)
Also tried turning on/off http2 on the output with no luck

Any ideas on getting it running. My Pi4 is running latest (thanks to watchtower), so I know it runs on ARM

**Comment by allendema** [comment]
Seems like a networking issue. Test it with default compose file.
https://github.com/searxng/searxng-docker/blob/master/docker-compose.yaml
